drafts/use-cases/5.creating-new-event.md

```diff
@@ -22,25 +23,26 @@ var client = new HydraClient();
 var operation = client.get("http://example.com")
     .getApiDocumentation()
     .getEntryPoint()
-    .getOperations()
-    .filter(op => op.method == "POST" && op.expects == "Event")[0];
+    .onLink('#events')
```

Comment:
I could do that. @alien-mcl when you introduced that naming, did `get` prefix supposed to suggest async function or you didn't intend any naming convention for sync vs. async?